rest/route/host.go

```diff
@@ -167,73 +160,60 @@ func hostPaginator(key string, limit int, args interface{}, sc data.Connector) (
 		}
 	}
 
-	tasks, err := sc.FindTasksByIds(taskIds)
+	tasks, err := hgh.sc.FindTasksByIds(taskIds)
```

Comment:
It was like this before, but this should only look for tasks if taskIds is not empty

Reply:
I don't really want to change behavior on this. At the same time, this feels like an implementation detail of the method on the data connector, rather than the route. At least to me.